user.py

```diff
@@ -1,25 +1,36 @@
 class User():
     def __init__(self):
-        self.email_password = {}
-        # self.email_IDname = {}
 
-    def validate(self ,email,password):
+        self.users_list = []
+        self.userid_password = {}
+        self.email_userid = {}
+        self.userid_username = {}
+        self.userid = 0
```

Comment:
Keep your naming conventions consistent. Rename this one to `id_sequence_counter` as well.